API/MoodisAPI/apiMoodis/apiMoodis/Controllers/EmotionController.cs

```diff
@@ -0,0 +1,95 @@
+﻿using apiMoodis.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Web.Http;
+using System.Data.Entity;
+
+namespace apiMoodis.Controllers
+{
+    public class EmotionController : ApiController
+    {
+        public HttpResponseMessage Get()
+        {
+            using (DatabaseContext dbContext = new DatabaseContext())
+            {
+                var emotions = dbContext.Emotions.ToList();
+                if (emotions.Count() == 0)
+                {
+                    return Request.CreateErrorResponse(HttpStatusCode.NotFound, "There are no emotions");
+                }
+                else
+                {
+                    return Request.CreateResponse(HttpStatusCode.OK, emotions);
```

Comment:
Standard practice is to return `IActionResult<T>` or `IActionResult`. This way, you can simply write
```
return emotions;
// Or if you want, return Ok(emotions);
```
This makes your response more explicit, and the code much cleaner.

The response will automatically be serialized into JSON

Reply:
However, since I noticed later on that your app is .NET Standard, not Core, those things are not applicable :/

This might help you then:
https://docs.microsoft.com/en-us/aspnet/web-api/overview/getting-started-with-aspnet-web-api/action-results#ihttpactionresult